docs/Using-Fleet/Vulnerability-Processing.md

```diff
@@ -26,6 +27,41 @@ Vulnerability processing happens on the Fleet instance and not on the host machi
 detected vulnerabilities cannot be used in the same way you would use an osquery query (e.g. you wouldn't
 be able write a query to retrieve all CVEs).
 
+## Coverage
+
+Fleet attempts to detect vulnerabilities for installed software that falls into the following categories (types):
+
+### Apps
+- macOS
+- Windows
+
+### Browser plugins
+- macOS, Windows, Linux
+  - Chrome extensions
+  - Firefox extensions
+
+### Packages
+- macOS
+  - Python
+  - Packages installed using Homebrew
+
+- Windows
+  - Python
+  - Atom
+  - Packages installed using Chocolatey
+
+- Linux
+  - deb
+  - RPM
+  - Atom
+  - Python
+  - Packages installed using APT (Debian)
```

Comment:
This is the same as `deb`.

Reply:
The only reason "Packages installed using YUM (Red Hat)" is in here is because we thought the query Fleet runs to collect software includes the `apt_sources` table. This is not the case.

Reply:
`apt_sources` lists the repositories APT uses, not the packages.